Mathlib/RingTheory/JacobsonIdeal.lean

```diff
@@ -214,6 +225,44 @@ theorem jacobson_mono {I J : Ideal R} : I ≤ J → I.jacobson ≤ J.jacobson :=
   erw [mem_sInf] at hx ⊢
   exact fun K ⟨hK, hK_max⟩ => hx ⟨Trans.trans h hK, hK_max⟩
 
+/-- The Jacobson radical of a two-sided ideal is two-sided.
+
+It is preferable to use `TwoSidedIdeal.jacobson` instead of this lemma. -/
+theorem jacobson_mul_mem_right {I : Ideal R}
+    (mul_mem_right : ∀ {x y}, x ∈ I → x * y ∈ I) :
+    ∀ {x y}, x ∈ I.jacobson → x * y ∈ I.jacobson := by
```

Comment:
I found that [this unmerged mathlib3 PR](https://github.com/leanprover-community/mathlib3/pull/13862/files#diff-d50d7dee9b427f9710e98cc536adfd95f656e65f85f1ac179cd465c399375551R128-R129) actually contains this result (in the special case `I = 0`, but the proof probably generalizes, or else consider the quotient R/I).

Reply:
Do you think anything from that PR should be added to mathlib4, or are we happy with the API added here?

Reply:
I'll work on a proof that IsArtinianRing implies IsNoetherianRing starting tomorrow; we'll see what else I need.